pkg/features/kube_features.go

```diff
@@ -898,6 +898,13 @@ const (
 	// Enables user namespace support for stateless pods.
 	UserNamespacesSupport featuregate.Feature = "UserNamespacesSupport"
 
+	// owner: @mattcarry, @sunnylovestiramisu
+	// kep: https://kep.k8s.io/3751
+	// alpha: v1.29
+	//
+	// Enables user specified volume attributes for persistent volumes, like iops and throughput.
+	VolumeAttributesClass featuregate.Feature = "VolumeAttributesClass"
```

Comment:
If needed at all, "like iops and throughput" might be better put on the API declaration. Here it might be better to state what resources are allowed to do what with this feature?